Sources/Build/BuildDescription/ProductBuildDescription.swift

```diff
@@ -243,6 +246,16 @@ public final class ProductBuildDescription: SPMBuildCore.ProductBuildDescription
             throw InternalError("unexpectedly asked to generate linker arguments for a plugin product")
         }
 
+        if let resourcesPath = self.buildParameters.toolchain.swiftResourcesPath(isStatic: isLinkingStaticStdlib) {
+            args += ["-resource-dir", "\(resourcesPath)"]
```

Comment:
@MaxDesiatov is it correct to rely on interpolation here instead of `pathString`?

Reply:
should this just move into the `if self.buildParameters.shouldLinkStaticSwiftStdlib` branch? its used only there it seems

Reply:
Interpolation will render it properly on Windows: https://github.com/apple/swift-package-manager/blob/14ff299060cfd7f1bab3a4fc9065f77d427f8093/Sources/Basics/FileSystem/NativePathExtensions.swift#L16